### PR TITLE
feat: check template command

### PR DESCRIPTION
**Pull request by jonas-martinez**
Closes https://github.com/lenra-io/lenra_cli/issues/51 #59 

## Checklist
- [ ] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [ ] I included unit tests that cover my changes
- [x] I added/updated the documentation about my changes
- [x] I made my own code-review before requesting one

### CODE REVIEW

File: src/cli/check/mod.rs [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+    #[clap(subcommand)]
+    command: CheckCommandType,
+}
+
+/// The check subcommands
+#[derive(Subcommand, Clone)]
+pub enum CheckCommandType {
+    // /// Check the current project as an app
+    // App(CheckParameters),
+    /// Check the current project as a template
```
Comment (jonas-martinez) [inline src/cli/check/mod.rs:30]: Add documentation to explain how to do the check (start the application using --expose)

Comment (jonas-martinez) [reply src/cli/check/mod.rs:30]: @taorepoara

Comment (taorepoara) [reply src/cli/check/mod.rs]: It's already documented: `lenra check --help`

**Review by taorepoara — APPROVED** [review]

File: docs/build.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+$ lenra build --help
+lenra-build 
+Build your app in release mode
+
+USAGE:
+    lenra build [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+        --expose <EXPOSE>    Exposes services ports [default: app postgres mongo] [possible values:
```
Comment (jonas-martinez) [inline docs/build.md:16]: Does this mean that not setting the `--expose` parameter will result into the CLI assigning by default the `--expose app postgres mongo` ?

File: docs/dev.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+$ lenra dev --help
+lenra-dev 
+Start the app in an interactive mode
+
+USAGE:
+    lenra dev [OPTIONS]
+
+OPTIONS:
+        --config <CONFIG>    The app configuration file [default: lenra.yml]
+        --expose <EXPOSE>    Exposes services ports [default: app postgres mongo] [possible values:
```
Comment (jonas-martinez) [inline docs/dev.md:17]: Same question here

File: docs/build.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# build
+
+This subcommand builds the Lenra app of the current directory.
+The app configuration are defined by a [configuration file](#configuration-file).
```
Comment (jonas-martinez) [inline docs/build.md:4]:
```suggestion
The app configuration is defined by a [configuration file](#configuration-file).
```

**Review by jonas-martinez — COMMENTED** [review]
When running the check on template-node I get this error:

```console
manifest            : Warning
    manifest:additionalProperty:manifest.listeners
        Additional property manifest.listeners
    manifest:additionalProperty:manifest.views
        Additional property manifest.views
view:main           : Error
    view:main:missingProperty:children
        Missing property children
    view:main:missingProperty:crossAxisAlignment
        Missing property crossAxisAlignment
    view:main:missingProperty:direction
        Missing property direction
    view:main:missingProperty:scroll
        Missing property scroll
    view:main:missingProperty:spacing
        Missing property spacing
    view:main:missingProperty:type
        Missing property type
    view:main:additionalProperty:view
        Additional property view
view:menu           : Error
    view:menu:missingProperty:child
        Missing property child
    view:menu:missingProperty:decoration
        Missing property decoration
    view:menu:missingProperty:padding
        Missing property padding
    view:menu:missingProperty:type
        Missing property type
    view:menu:additionalProperty:view
        Additional property view
view:home           : Error
    view:home:missingProperty:children
        Missing property children
    view:home:missingProperty:crossAxisAlignment
        Missing property crossAxisAlignment
    view:home:missingProperty:direction
        Missing property direction
    view:home:missingProperty:mainAxisAlignment
        Missing property mainAxisAlignment
    view:home:missingProperty:spacing
        Missing property spacing
    view:home:missingProperty:type
        Missing property type
    view:home:additionalProperty:view
        Additional property view
view:counter        : Error
    view:counter:missingProperty:children
        Missing property children
    view:counter:missingProperty:crossAxisAlignment
        Missing property crossAxisAlignment
    view:counter:missingProperty:mainAxisAlignment
        Missing property mainAxisAlignment
    view:counter:missingProperty:spacing
        Missing property spacing
    view:counter:missingProperty:type
        Missing property type
    view:counter:additionalProperty:view
        Additional property view
Error: Check
```

File: docs/build.md (hunk 2 above)
Comment (taorepoara) [reply docs/build.md:16]: No, what I wanted is that if the expose is not set, the expose is empty, but if you use the expose option without value, the app, mongo and postgres services are exposed.

This is not working yet, I'm working on it.

File: docs/dev.md (hunk 3 above)
Comment (taorepoara) [reply docs/dev.md:17]: Same answer :stuck_out_tongue:

**Comment by taorepoara** [comment]
> When running the check on template-node I get this error:
> [...]

The node template is outdated. This might be normal.

File: docs/build.md (hunk 2 above)
Comment (taorepoara) [reply docs/build.md]: I have just pushed the fix

**Comment by taorepoara** [comment]
I have remove the Windows ARM 64 arch since it fails in the pipeline and nearly nobody use it. We can re-add it later

**Review by taorepoara — APPROVED** [review]

**Pull request merged**